conda_smithy/feedstock_content/ci_support/upload_or_check_non_existence.py

```diff
@@ -44,7 +44,7 @@ def upload(cli, meta, owner, channels):
     try:
         with open('binstar.token', 'w') as fh:
             fh.write(cli.token)
-        subprocess.check_call(['binstar', '--quiet', '-t', 'binstar.token',
+        subprocess.check_call(['anaconda-client', '--quiet', '-t', 'binstar.token',
```

Comment:
Should this be only anaconda as the change in https://github.com/conda-forge/obvious-ci-feedstock/pull/3/files#r43991411?